lib/accordion/readMe.md

```diff
@@ -0,0 +1,50 @@
+## Accordion
+
+A basic accordion component.
+
+### Properties
+| Name | Type | Default | Description |
+|:-----|:-----|:-----|:-----|
+| `data` | `Array` | &nbsp; | An array of contents for accordion. ( `Required` ) |
+| `headerKey` | `String` | `header` | Set a Key for the header of the accordion. | 
+| `contentKey` | `String` | `content` | Set a Key for the content of the accordion.|
```

Comment:
It actually doesn't set the `key`, It specifies what `key` to be used from user given data. Please add a statement accordingly.

Reply:
done! https://github.com/Codebrahma/react-lite-ui/pull/327/commits/afbc1ac43e046a808304e3385490e406de996c86